_config.yml

```
@@ -1081,4 +1081,10 @@ podcast:
       file_byte_length: "32892977"
       duration: "00:34:07"
       credits: "Jon Allured, Pavlos Vinieratos, Devon Blandin"
- 
\ No newline at end of file
+     - title: "Interview with Artsy's Former CTO Daniel Doubrovkine - Part 1"
```

```suggestion
     - title: "15: Interview with Artsy's Former CTO Daniel Doubrovkine - Part 1"
```

I'm sure there's a way to automate this so that it's not reliant on PR review, but I really like having episodes numbered.

---

> I can't wait to listen to this interview!!
> 
> One quick thing I noticed -- I think db's track might have gotten nudged a bit out of line with yours. There's a noticeable gap _after_ db talks and before you start, and it sounds like he's interrupting you when he starts talking.
> 
> Sound quality & levels sound great, though!

OH NO i thought i fixed that. thanks for catching. will take another look

---

and will add `15` to the above!